### PR TITLE
community: sambastudio llms api v2 support

### PR DESCRIPTION
**Pull request by jhpiedrahitao**
- **Description:** SambaStudio GenericV2 API support

### CODE REVIEW

**Comment by snova-jorgep** [comment]
@baskaryan can you take a look at this, Thanks!

**Comment by jhpiedrahitao** [comment]
@eyurtsev can you take a look at this, Thanks!

**Comment by snova-jorgep** [comment]
@ccurme can you take a look at this please

File: libs/community/langchain_community/llms/sambanova.py [hunk 1]
```diff
@@ -535,14 +535,15 @@ def _process_streaming_response(
...
+            "api/v2/predict/generic" in self.api_base_uri
+            or "api/predict/generic" in self.api_base_uri
+        ):
             try:
                 for line in response.iter_lines():
                     chunk = json.loads(line)
                     if "status_code" not in chunk:
                         chunk["status_code"] = response.status_code
-                    if chunk["status_code"] == 200 and chunk.get("error"):
-                        chunk["result"] = {"responses": [{"stream_token": ""}]}
```
Comment (ccurme) [inline libs/community/langchain_community/llms/sambanova.py:545]: is this breaking for v1?

**Review by ccurme — APPROVED** [review]

**Pull request merged**